repos/system_upgrade/el7toel8/libraries/dnfplugin.py

```diff
@@ -174,7 +174,7 @@ def perform_transaction_install(target_userspace_info, storage_info, used_repos,
         '/proc:/installroot/proc',
         '/run/udev:/installroot/run/udev'
     ]
-    already_mounted = set([entry.split(':')[0] for entry in bind_mounts])
+    already_mounted = {[entry.split(':')[0] for entry in bind_mounts]}
```

Comment:
this will crash during runtime. Original solurion seems ok. If you want to use explicitly set comprehension, ommit the list comprehension first:
```suggestion
    already_mounted = {entry.split(':')[0] for entry in bind_mounts}
```